sympy/core/sympify.py

```diff
@@ -66,6 +66,9 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
        - booleans, including ``None`` (will leave ``None`` unchanged)
        - lists, sets or tuples containing any of the above
 
+    Note that this function uses ``eval``, and thus shouldn't be used on
```

Comment:
@asmeurer @moorepants  I'm new to Sympy...what's the process for handling this? Do I first create an issue around it to link the PR to or should I just create a PR and reference this issue?